changelogs-sdk.gradle

```diff
@@ -247,7 +247,7 @@ buildChangelogs {
 			tickets.each {
 				String ticket ->
 
-				text += "\n[" + ticket + "](https://issues.liferay.com/browse/" + ticket + ")"
+				text += "\n[" + ticket + "]: https://issues.liferay.com/browse/" + ticket
```

Comment:
Hey @ling-alan-huang @petershin, 

I think we need to make this change in `liferay-portal` too.

This is because `[Link display Text](https://www.liferay.com)` shows the link inline. If you look at the https://github.com/liferay/liferay-portal/blob/master/modules/sdk/gradle-plugins-workspace/CHANGELOG.markdown#markdown-links you'll see the result.

After I regenerated after this change, everything looks good on our end.